config.go

```diff
@@ -125,10 +125,11 @@ type Config struct {
 	// Maximum number of times this consumer will attempt to process a message before giving up
 	MaxAttempts uint16 `opt:"max_attempts" min:"0" max:"65535" default:"5"`
 
-	// Duration to wait for a message from a producer when in a state where RDY
-	// counts are re-distributed (ie. max_in_flight < num_producers)
+	// Duration to wait for a message from an nsqd when in a state where RDY
+	// counts are re-distributed (e.g. max_in_flight < num_producers)
 	LowRdyIdleTimeout time.Duration `opt:"low_rdy_idle_timeout" min:"1s" max:"5m" default:"10s"`
-
+	// Duration to wait until redistributing RDY for an nsqd regardless of LowRdyIdleTimeout
+	LowRdyTimeout time.Duration `opt:"low_rdy_timeout" min:"1s" max:"5m" default:"30s"`
```

Comment:
This should probably never be lower than `LowRdyIdleTimeout`, I can enforce this...